src/backend/distributed/citus--7.0-1.sql

```diff
@@ -0,0 +1,1431 @@
+
+-- define shard repair function
+CREATE FUNCTION master_copy_shard_placement(shard_id bigint,
+                                            source_node_name text,
+                                            source_node_port integer,
+                                            target_node_name text,
+                                            target_node_port integer)
+RETURNS void
+AS 'MODULE_PATHNAME'
+LANGUAGE C STRICT;
+
+RESET search_path;
+/* citus--5.0--5.0-1.sql */
+
+GRANT SELECT ON pg_catalog.pg_dist_partition TO public;
+GRANT SELECT ON pg_catalog.pg_dist_shard TO public;
+GRANT SELECT ON pg_catalog.pg_dist_shard_placement TO public;
+/* citus--5.0-2--5.1-1.sql */
+
+/* empty, but required to update the extension version */
+CREATE FUNCTION pg_catalog.master_modify_multiple_shards(text)
+    RETURNS integer
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_modify_multiple_shards$$;
+COMMENT ON FUNCTION master_modify_multiple_shards(text)
+    IS 'push delete and update queries to shards';
+
+CREATE OR REPLACE FUNCTION pg_catalog.master_update_shard_statistics(shard_id bigint)
+    RETURNS bigint
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_update_shard_statistics$$;
+COMMENT ON FUNCTION master_update_shard_statistics(bigint)
+    IS 'updates shard statistics and returns the updated shard size';
+DROP FUNCTION IF EXISTS pg_catalog.worker_apply_shard_ddl_command(bigint, text);
+
+CREATE OR REPLACE FUNCTION pg_catalog.worker_apply_shard_ddl_command(bigint, text, text)
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$worker_apply_shard_ddl_command$$;
+COMMENT ON FUNCTION worker_apply_shard_ddl_command(bigint, text, text)
+    IS 'extend ddl command with shardId and apply on database';
+DROP FUNCTION IF EXISTS pg_catalog.worker_fetch_foreign_file(text, bigint, text[], integer[]);
+
+CREATE OR REPLACE FUNCTION pg_catalog.worker_fetch_foreign_file(text, text, bigint, text[], integer[])
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$worker_fetch_foreign_file$$;
+COMMENT ON FUNCTION pg_catalog.worker_fetch_foreign_file(text, text, bigint, text[], integer[])
+    IS 'fetch foreign file from remote node and apply file';
+
+DROP FUNCTION IF EXISTS pg_catalog.worker_fetch_regular_table(text, bigint, text[], integer[]);
+
+CREATE OR REPLACE FUNCTION pg_catalog.worker_fetch_regular_table(text, text, bigint, text[], integer[])
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$worker_fetch_regular_table$$;
+COMMENT ON FUNCTION pg_catalog.worker_fetch_regular_table(text, text, bigint, text[], integer[])
+    IS 'fetch PostgreSQL table from remote node';
+CREATE OR REPLACE FUNCTION pg_catalog.worker_apply_shard_ddl_command(bigint, text)
+    RETURNS void
+    LANGUAGE sql
+AS $worker_apply_shard_ddl_command$
+    SELECT pg_catalog.worker_apply_shard_ddl_command($1, 'public', $2);
+$worker_apply_shard_ddl_command$;
```

Comment:
These seem like some wrappers were introduced for backwards compatibility when schema support was introduced (for when the workers are ahead of the coordinator). Those wrappers can be removed.

Reply:
I've removed the previous 2-argument variant of this function, but the wrapper should be removed in a separate migration in 8.3. There should be no difference in schema between upgrading to `7.0-1` or installing to `7.0-1` through this script